docs/RELEASE_WORKFLOW.md

```diff
@@ -1,36 +1,59 @@
-# Release Workflow
+# How to do releases
+
+## Test releases
+
+* Before every official release we make a test release, similar to "Release
```

Comment:
at-least one, most of the time multiple until we feel like it's stable enough for releasing.
Most of the time we said sth like when the release candidate produced/revealed no blocking issues in one week.